### ISSUE TITLE
ToDo List only and Sorting

### ISSUE THREAD

**Issue by P3lUZa** [mank319/Go-For-It#64]
When you add the option to use it as "ToDo list only" it would be great to add tasks in reverse order, so instead of putting new notes at the bottom put them at the top. That would be awesome!

**Comment by mank319** [comment]
I agree that this is an issue, as I often find myself moving tasks to the top right after adding them. However adding them to the bottom of the list makes more sense when noting multiple tasks in chronological order. Moreover I consider it the most intuitive default behaviour as it goes with the user's "reading flow".

What is your opinion on the following solution @P3lUZa? `Return` adds a task to the bottom of the list but pressing `Ctrl+Return` adds it to the top.

**Comment by P3lUZa** [comment]
That sounds good :smiley:

**Comment by P3lUZa** [comment]
However,  I think it would be better to make it an option (under settings) so users can decide whether they want return to add tasks at the bottom and ctrl-return at the top or the other way.

The reason I am so insistent about putting new tasks at the top is because i believe there are two kinds of tasks. The ones that you are gonna do "right now" and "the others". Imagine you have 200 tasks of "the others" category and you want to add 10 new tasks, if those 10 task are of "the right now" category, then you want to open the app and see those tasks instead of having to scroll the 200 other tasks to discover those 10 tasks at the end. And you would have to do that always since your "right now" tasks would always be at the bottom. :yum:
